### DISCUSSION TITLE
Newsletter

### DISCUSSION THREAD

**Discussion by wipfli** [maplibre/maplibre-navigation-ios#39] [Q&A]
Hi everyone. It is great to see that there is more activity in the navigation repo again!

Would anyone be interested in writing one or two paragraphs for the MapLibre Newsletter about the current ongoing efforts?

**Comment by boldtrn** [comment]
Thanks for bringing this up, I can write a few sentences. Are there any requirements / deadline?

**Reply by wipfli** [reply]
Great! There are no strict requirements. The deadline is maybe in 3 days, would that work for you?

**Reply by boldtrn** [reply]
Sorry for the delay. Here would be my proposal of the writeup. Please feel free to alter the text as needed :). I also added some shameless self advertisement ;), feel free to remove this if this does not fit.

We are currently reviving maplibre-navigation-ios. After we have improved maplibre-navigation-android recently, we are currently doing the same for iOS. We have updated dependencies like maplibre-native to version 6 with Metal support and use SPM instead of CocoaPods for dependency management. We have added hooks to make customizing some of the navigation functions possible. Kurviger is currently BEAT testing their app with maplibre-navigation-ios, if you would like to give it a spin, you can try the app for free [here](https://testflight.apple.com/join/Y2eMnS3D) - please note, only the navigation is using maplibre-navigation-ios. In the future we also want to make it easier to customize the out of the box navigation UI and to move the Android and iOS navigation libraries closer together. Would you like to join the effort? PRs and contributions are welcome.

**Reply by wipfli** [reply]
Very good timing, I was just about to start putting the newsletter together! Thanks for the paragraph, @boldtrn. I will copy it over to https://github.com/maplibre/maplibre.github.io/issues/291
